PDP supports videos OOTB (#741)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/product-details/index.mdx

The PDP component provides a variety of fully-customizable controls to showcase your products according to your brand's aesthetics and build interactive experiences that engage customers. Use cases include:

> - **Image Carousel**: Display multiple product images and videos in a carousel format, allowing users to view different angles, details, and media of the product.

Reviewer: ```suggestion
- **Image Carousel**: Display multiple product images and videos in a carousel format, allowing users to browse product images, videos, and other details.
```
